### PR TITLE
[DomainTools.xml] Rewrote ruleset

### LINKED ISSUE #3085: Activate Domain Tools ruleset
There is already d [domain tools ruleset](https://github.com/EFForg/https-everywhere/blob/bcb101723edb31a933b1f4e26cce17c2364da31d/src/chrome/content/rules/DomainTools.xml), but it does neither show up in HTTPS-E 5.1.1* in my case nor is it activated in the XML by default.

However [Domain tools](https://www.domaintools.com/) seems to support HTTPS (at least on [a few sites](https://whois.domaintools.com/) - did not tested it everywhere). 

\* Okay, I got the issue why it does not show up: The ww subdomain is missing.

### PR DESCRIPTION
**Pull request by rugk**
This addresses https://github.com/EFForg/https-everywhere/issues/3085 a bit.
Anyway I think more should be done here. I don't even know if `https://secure.domaintools.com` is used anymore. At least it now redirects to http://research.domaintools.com/ which is a bit strange.
Also there are much more subdomains included in the certificate:

`www.domaintools.com domaintools.com api.domaintools.com assets.dt-static.com brandmonitor.domaintools.com directory.domaintools.com domainreport.domaintools.com domaintools.com freeapi.domaintools.com images.dt-static.com img.domaintools.com marketplace.domaintools.com reverseip.domaintools.com reversens.domaintools.com reversewhois.domaintools.com source.domaintools.com thumbnails.domaintools.com whitelabel.domaintools.com whois.domaintools.com yahoo.domaintools.com`

----
Fixes  #3085

### CODE REVIEW

**Comment by rugk** [comment]
Okay, I think a better approach than adding a bunch of test URLs now, would be to get rid of this wildcard there...

**Comment by rugk** [comment]
With this update it now fixes #3085.

**Comment by rugk** [comment]
Any suggestions or can this be merged?

**Comment by semenko** [comment]
Looks like this now has a conflict. Happy to merge if you resolve it! :+1:

**Comment by rugk** [comment]
Is this okay?

**Comment by semenko** [comment]
Travis failure :( -- apparently a duplicate ruleset exists?

**Comment by rugk** [comment]
Ah okay. This way now?

**Comment by semenko** [comment]
Probably better to see what ruleset contains the support subdomain, and if it could be merged into one Domaintools rule

**Comment by rugk** [comment]
I do not think this can be merged. Because [this](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/DomainTools.com-falsemixed.xml) is the other ruleset and it seems it has it's reason to be separate (mixed content). This is also indicated in the comments of this ruleset.

**Comment by rugk** [comment]
The Travis test is successful. So is there anything else which should be changed?

File: src/chrome/content/rules/DomainTools.xml [hunk 1]
```diff
@@ -1,124 +1,70 @@
...
+            - assets
+            - images
+            - img
 -->
-<ruleset name="DomainTools.com (partial)">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="domaintools.com" />
-	<target host="account.domaintools.com" />
```
Comment (semenko) [inline src/chrome/content/rules/DomainTools.xml]: Is there a reason account.domaintools.com is missing from the new set?

Comment (rugk) [inline src/chrome/content/rules/DomainTools.xml]: No, I'll add it again. Thanks for letting me know.

**Pull request merged**